src/browser/navigation-panel.js

```diff
@@ -78,15 +91,19 @@ const NavigationControls = Component(({input, tabStrip, webViewer, title}) =>
     ]),
     DOM.div({key: 'reload-button',
              className: 'reloadbutton',
+             style: {color: theme.buttons},
```

Comment:
I'd do `style: theme.reloadButton` instead of `style: theme.button` so it's up to `readTheme` to decide if all buttons share same style or some of them are special.